### DISCUSSION TITLE
The sample data of gemgis

### DISCUSSION THREAD

**Discussion by gisqhd** [cgre-aachen/gemgis#201] [General]
The sample data of gemgis is only 1-24. Where can I download all the data（1-53）?

**Comment by AlexanderJuestel** [comment]
Hello @gisghd,

I could be that the data is not available yet for download. What exactly do you need? Maybe I can help you out!

Cheers Alex

**Comment by AlexanderJuestel** [comment]
Closed due to inactivity
